code/modules/awaymissions/mission_code/ruins/moonoutpost19.dm

```diff
@@ -0,0 +1,37 @@
+//papers for moonoutpost19.dmm
+
+/obj/item/paper/fluff/ruins/moonoutpost19/lobby
+	name = "a note to Joe"
+	info = "<br>Hey Joe, since you keep forgetting here's the list of lockdown areas and the locations of control buttons. Thank me later.<br><br><b>(Lobby)</b><br><br>- Research Division lockdown<br>- Medical Ward lockdown<br>- Cargo Bay gates<br><br><b>(Cargo Bay)</b><br><br>- Exterior hangar gate<br>- Cargo bay gates<br><br><b>(Senior Researcher's Office)</b><br><br>- Specimen Containtment Zone lockdown<br><br>Note: keep in mind, in case of power outages these buttons won't be functional. In this case you'll find a few generators in engineering wing, set them up properly then report to technicians ASAP."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/entrance
+	name = "hastily written note"
+	info = "A warning to whose reading this, things here went really wrong. Way too damn wrong. Turn back while you can, there's nothing left for anyone. Not anymore."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/rockstarcop
+	name = "eerie note"
+	info = "(you see some unintelligible scribbles covered in blood, only words you can choose are...) <br><br>If everybody knew, I never knew <br>I will wake up in a new life<br> Down by the seaside..."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/engineering
+	name = "emergency procedures"
+	info = "(most of the parts are unreadable due to stains on the paper, the only words you can make out is...)<br><br> ...incase of power outages, use the provided portable generators. It is recommended that setting up two of them at once rather than working with one generator on high power levels. It is also recommended using the pre-marked spots for generator setup. If you are uncertain where it is, use a t-ray scanner to check the floors. Provided plasma sheets will suffice until maintai..."
```

Comment:
```suggestion
	info = "(most of the parts are unreadable due to stains on the paper, the only words you can make out is...)<br><br> ...in case of power outages, use the provided portable generators. It is recommended setting up two of them at once rather than working with one generator on high power levels. It is also recommended using the pre-marked spots for generator setup. If you are uncertain where it is, use a t-ray scanner to check the floors. Provided plasma sheets will suffice until maintai..."
```